brave/browser/resource_coordinator/guest_tab_manager.h

```diff
@@ -32,6 +32,11 @@ class GuestTabManager : public TabManager {
  public:
   GuestTabManager();
 
+  content::WebContents* CreateNullContents(
```

Comment:
There is no need for these two APIs to be public

Reply:
`CreateNullContents()` and `DestroyNullContents()` were called in `TabManager::DiscardWebContentsAt()`, but now they are called in `TabLifecycleUnitSource::TabLifecycleUnit::Discard()`. We kept the methods in `TabManager` but made them public.

Reply:
don't make any changes to this code - I am working on it